src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -499,7 +499,7 @@ void Deconvolution::getSupportedDescriptors() {
 
             return AclDeconvExecutorBuilder::customIsSupported(deconvAttrs, srcMemoryDescs, dstMemoryDescs);
         };
-        useACL = checkDesc(LayoutType::nspc) || checkDesc(LayoutType::ncsp);
+        useACL = checkDesc(LayoutType::ncsp);
```

Comment:
What is the reason? It should drop the perf

Reply:
nspc don't work for all deconvolution cases, because we don't separate executor and now validate function in acl for ncsp always return false